single-line-diagram-core/src/test/resources/TestCase12GraphWithNodesInfosTopological.svg

```diff
@@ -278,7 +278,7 @@ polyline {fill: none}
             </g>
```

Comment:
The fact that you needed to change this test reference shows that your modifications introduced some regressions. The feeders in the test are not disconnected.